inventories/broker.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import yaml
+import os
+import subprocess
+import sys
+
+try:
+    from StringIO import StringIO  # pyright: reportMissingImports=false
+except ImportError:
+    from io import StringIO  # pyright: reportMissingImports=false
+
+from collections import defaultdict
+
+
+try:
+    DEVNULL = subprocess.DEVNULL
+except AttributeError:
+    DEVNULL = open(os.devnull, 'w')
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description="Broker inventory script")
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument('--list', action='store_true')
+    group.add_argument('--host')
+    return parser.parse_args()
+
+
+def get_running_hosts():
+    try:
+        subprocess.check_call(["which", "broker"], stdout=DEVNULL)
+    except subprocess.CalledProcessError:
+        return
+
+    cmd = "broker inventory --details"
+    output = subprocess.check_output(cmd.split(), universal_newlines=True).rstrip()
```

Comment:
I'd write out the list instead of splitting a string, even if in practice it probably doesn't matter
```suggestion
    cmd = ["broker", "inventory", "--details"]
    output = subprocess.check_output(cmd, universal_newlines=True).rstrip()
```